src/db_api_util.ts

```diff
@@ -591,6 +796,22 @@ export async function logVolunteerVoterClaimToDb(
   }
 }
 
+export async function getVoterHasVolunteer(
+  userId: string,
+  userPhoneNumber: string
+): Promise<boolean> {
+  const client = await pool.connect();
+  try {
+    const result = await client.query(
+      'SELECT * FROM volunteer_voter_claims WHERE user_id = $1 AND user_phone_number = $2',
```

Comment:
nit: you shouldn't need to check both user_id and user_phone_number. doesn't hurt if you prefer it though. if you choose one go with user_id